docs/source/installation.md

```diff
@@ -73,21 +73,46 @@ It should download a pretrained model then print something like
 
 ## Installing from source
 
-To install from source, clone the repository and install with the following commands:
+Note that the following will will install not the latest released version, but the bleeding edge `master` version, which may be not-quite ready for normal usage, but you may want to use it in case a bug has been fixed since the last official release and a new release hasn't yet been rolled out.
+
+```
+pip install git+https://github.com/huggingface/transformers
+```
+
+Again, you can run
+
+```bash
+python -c "from transformers import pipeline; print(pipeline('sentiment-analysis')('I hate you'))"
+```
+
+to check 🤗 Transformers is properly installed.
+
+## Editable install
+
+If you are an advanced user who wants to constantly use the bleeding edge `master` version of the source code, the easiest way to do it is via an editable install. This is done by cloning the repository and installing with the following commands:
```

Comment:
```suggestion
If you want to constantly use the bleeding edge `master` version of the source code, or if you want to contribute to the library and need to test the changes in the code you're making, you will need an editable install. This is done by cloning the repository and installing with the following commands:
```
I suggest removing the "advanced user" mention as people could then self-censure when they do need a source install, and adding a mention about the contribution.